packages/fluentui/react-northstar/src/themes/teams/colors.ts

```diff
@@ -456,6 +462,7 @@ export const colorScheme: ColorSchemeMapping<ColorScheme, TeamsColorNames> = {
     foreground: colors.yellow[300],
     foreground1: colors.grey[800],
     foreground2: colors.white,
+    foreground3: colors.grey[750],
```

Comment:
Did you verify with Kay/Daisy that we should be adding values into the color schemes?

Reply:
yes